.github/requirements.txt

```diff
@@ -0,0 +1,3 @@
+tutor[full]<16
+setuptools
+git+https://github.com/hastexo/tutor-contrib-s3@v1.4.0
```

Comment:
Please check the tutor contrib S3 [compatibility matrix](https://github.com/hastexo/tutor-contrib-s3?tab=readme-ov-file#version-compatibility-matrix)